src/ui/public/notify/notifier.js

```diff
@@ -318,48 +320,69 @@ Notifier.prototype._showFatal = function (err) {
  * @param  {Error|String} err
  * @param  {Function} cb
  */
-Notifier.prototype.error = function (err, cb) {
-  return add({
+Notifier.prototype.error = function (err, opts, cb) {
+  const overrideable = ['lifetime', 'icon'];
+  if (_.isFunction(opts)) {
```

Comment:
I didn't see any in our code, but it's possible plugins use it. I figured it best not to break the API until we take a bigger stab at updating Notifications.